Burrow: parenthesise the buffer sizes the AWS patch enlarges.

Without parentheses, OPTION_PARM_SIZE + 16 expands to 1 << 33 (an int
overflow in options_warning_safe_ml) and USER_PASS_LEN - 1 to 1 << 16.

diff --git a/src/openvpn/buffer.h b/src/openvpn/buffer.h
index 3760175..66ac439 100644
--- a/src/openvpn/buffer.h
+++ b/src/openvpn/buffer.h
@@ -27,7 +27,7 @@
 #include "basic.h"
 #include "error.h"
 
-#define BUF_SIZE_MAX 1 << 21
+#define BUF_SIZE_MAX (1 << 21)
 
 /*
  * Define verify_align function, otherwise
diff --git a/src/openvpn/common.h b/src/openvpn/common.h
index 61ee72e..f3010bd 100644
--- a/src/openvpn/common.h
+++ b/src/openvpn/common.h
@@ -66,7 +66,7 @@ typedef unsigned long ptr_type;
  * maximum size of a single TLS message (cleartext).
  * This parameter must be >= PUSH_BUNDLE_SIZE
  */
-#define TLS_CHANNEL_BUF_SIZE 1 << 18
+#define TLS_CHANNEL_BUF_SIZE (1 << 18)
 
 /* TLS control buffer minimum size
  *
diff --git a/src/openvpn/error.h b/src/openvpn/error.h
index cb0c68e..e7fe796 100644
--- a/src/openvpn/error.h
+++ b/src/openvpn/error.h
@@ -37,7 +37,7 @@
 /*
  * Increase the error buffer size to 256 KB.
  */
-#define ERR_BUF_SIZE 1 << 18
+#define ERR_BUF_SIZE (1 << 18)
 #endif
 
 struct gc_arena;
diff --git a/src/openvpn/misc.h b/src/openvpn/misc.h
index 3e7cd3e..9724782 100644
--- a/src/openvpn/misc.h
+++ b/src/openvpn/misc.h
@@ -69,7 +69,7 @@ struct user_pass
 /*
  * Increase the username and password length size to 128KB.
  */
-#define USER_PASS_LEN 1 << 17
+#define USER_PASS_LEN (1 << 17)
 #endif
     /* Note that username and password are expected to be null-terminated */
     char username[USER_PASS_LEN];
